tools/src/icon4pytools/py2fgen/wrappers/diffusion_wrapper.py

```diff
@@ -30,8 +33,8 @@
 )
 from icon4py.model.common import dimension as dims, field_type_aliases as fa, settings
 from icon4py.model.common.constants import DEFAULT_PHYSICS_DYNAMICS_TIMESTEP_RATIO
-from icon4py.model.common.grid import geometry
-from icon4py.model.common.grid.icon import GlobalGridParams, IconGrid
+from icon4py.model.common.grid import geometry, icon
+from icon4py.model.common.grid.icon import GlobalGridParams
 from icon4py.model.common.grid.vertical import VerticalGrid, VerticalGridConfig
```

Comment:
I have not so far payed any attention to the imports. As we discussed last we started fixing them to avoid direct symbol imports in `model` whenever we come across, to comply with the style guide. But I leave that up to you. You might want to change it independently of your functional changes.

Reply:
I have tried to follow this convention in most places. It's definitely possible that this is not followed yet strictly everywhere in `icon4pytools` but I can address this in a follow-up PR.